.github/workflows/integration.yaml

```diff
@@ -27,51 +27,87 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  zig:
-    name: Zig
+  # This only ensures the existence of Zig's test binary, and exists in order to work hard to build on
```

Comment:
this sentence is awkward. Can this be simplified to "This builds a zig test binary only if the same version tag hasn't been build before. This saves time as we rarely update the zig version."?